test-storybooks/portable-stories-kitchen-sink/vue3/package.json

```diff
@@ -16,66 +16,66 @@
-    "@storybook/builder-webpack5": "portal:../../../code/builders/builder-webpack5",
-    "@storybook/codemod": "portal:../../../code/lib/codemod",
-    "@storybook/core": "portal:../../../code/core",
-    "@storybook/core-webpack": "portal:../../../code/lib/core-webpack",
-    "@storybook/csf-plugin": "portal:../../../code/lib/csf-plugin",
-    "@storybook/ember": "portal:../../../code/frameworks/ember",
-    "@storybook/html": "portal:../../../code/renderers/html",
-    "@storybook/html-webpack5": "portal:../../../code/frameworks/html-webpack5",
-    "@storybook/instrumenter": "portal:../../../code/lib/instrumenter",
-    "@storybook/nextjs": "portal:../../../code/frameworks/nextjs",
-    "@storybook/preact": "portal:../../../code/renderers/preact",
-    "@storybook/preact-vite": "portal:../../../code/frameworks/preact-vite",
-    "@storybook/preact-webpack5": "portal:../../../code/frameworks/preact-webpack5",
-    "@storybook/preset-create-react-app": "portal:../../../code/presets/create-react-app",
-    "@storybook/preset-html-webpack": "portal:../../../code/presets/html-webpack",
-    "@storybook/preset-preact-webpack": "portal:../../../code/presets/preact-webpack",
-    "@storybook/preset-react-webpack": "portal:../../../code/presets/react-webpack",
-    "@storybook/preset-server-webpack": "portal:../../../code/presets/server-webpack",
-    "@storybook/preset-svelte-webpack": "portal:../../../code/presets/svelte-webpack",
-    "@storybook/preset-vue3-webpack": "portal:../../../code/presets/vue3-webpack",
-    "@storybook/react": "portal:../../../code/renderers/react",
-    "@storybook/react-dom-shim": "portal:../../../code/lib/react-dom-shim",
-    "@storybook/react-vite": "portal:../../../code/frameworks/react-vite",
-    "@storybook/react-webpack5": "portal:../../../code/frameworks/react-webpack5",
-    "@storybook/server": "portal:../../../code/renderers/server",
-    "@storybook/server-webpack5": "portal:../../../code/frameworks/server-webpack5",
-    "@storybook/source-loader": "portal:../../../code/lib/source-loader",
-    "@storybook/svelte": "portal:../../../code/renderers/svelte",
-    "@storybook/svelte-vite": "portal:../../../code/frameworks/svelte-vite",
-    "@storybook/svelte-webpack5": "portal:../../../code/frameworks/svelte-webpack5",
-    "@storybook/sveltekit": "portal:../../../code/frameworks/sveltekit",
-    "@storybook/test": "portal:../../../code/lib/test",
-    "@storybook/vue3": "portal:../../../code/renderers/vue3",
-    "@storybook/vue3-vite": "portal:../../../code/frameworks/vue3-vite",
-    "@storybook/vue3-webpack5": "portal:../../../code/frameworks/vue3-webpack5",
-    "@storybook/web-components": "portal:../../../code/renderers/web-components",
-    "@storybook/web-components-vite": "portal:../../../code/frameworks/web-components-vite",
-    "@storybook/web-components-webpack5": "portal:../../../code/frameworks/web-components-webpack5",
+    "@storybook/addon-a11y": "file:../../../code/addons/a11y",
```

Comment:
should we change all portable stories kitchen sinks to use `link` instead of `portal`?

Reply:
Well, it's very inconvenient for whoever needs to work on these if they are `file:` protocol, so I figured if they aren't broke, don't fix em.

@yannbf is the person working on these most, so I'll defer the decision if we choose consistency over ease of maintenance, to him.

I'm ok with either option.

Reply:
I'm OK with whatever change